pkg/command/lint/sarif_format.go

```diff
@@ -0,0 +1,56 @@
+package lint
+
+import (
+	"io"
+
+	"github.com/owenrumney/go-sarif/sarif"
+	"github.com/pkg/errors"
+	"golang.stackrox.io/kube-linter/pkg/run"
+)
+
+// FormatSarifJSON implements common.SARIFFormat.
+// Must be used only with lint.Command because understands only run.Result as data parameter.
+func FormatSarifJSON(out io.Writer, data interface{}) error {
+	if res, ok := data.(run.Result); ok {
+		return formatSarif(out, res)
+	}
+	return errors.New("Blah") // TODO: message
+}
+
+func formatSarif(out io.Writer, result run.Result) error {
+	sarifReport, err := sarif.New(sarif.Version210)
+	if err != nil {
+		return err
+	}
+
+	// TODO: have kube-linter and github address somewhere in "more common" place for such strings.
+	sarifRun := sarif.NewRun("kube-linter", "https://github.com/stackrox/kube-linter")
+	sarifReport.AddRun(sarifRun)
+
+	for _, r := range result.Reports {
+		k8sObjectName := r.Object.GetK8sObjectName()
+		sarifLocation := sarif.NewLocation()
+		sarifLocation.LogicalLocations = append(sarifLocation.LogicalLocations,
+			sarif.NewLogicalLocation().WithName(k8sObjectName.Name).WithKind("k8s object name"),
```

Comment:
This looks kind of weird. Just "Object Name" or "Kubernetes Object Name"?

Reply:
Renamed to "Object Name".